README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+---
+title: "Yunshu Hu"
+---
```

Comment:
Actually, this isn't quite necessary and boarders on wrong. This is called YAML (we'll be talking about it later on in the course). It goes at the top of every .Rmd but is not meant to appear in .md files.